spec/lib/farscape/agent_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Farscape::Agent do
+  describe '#instance' do
+
```

Comment:
personally, `it` has newline before or it does not, else looks bad, but I'm going to delete them